### PR TITLE
LPS-152971 [CP 2.0] Translate keys in Overview Page w/ Subscription

### PR DESCRIPTION
**Pull request by claricebrasil**

### CODE REVIEW

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/components/Table/Pagination/index.js [hunk 1]
```diff
@@ -54,9 +55,13 @@ const TablePagination = ({
...
-		<p className="mb-4 mx-4 pagination-results">{`Showing ${
-			itemsPerPage * activePage + 1 - itemsPerPage
-		} to ${totalItems} of ${totalItems} entries.`}</p>
+		<p className="mb-4 mx-4 pagination-results">{`${i18n.translate(
+			'showing'
+		)} ${itemsPerPage * activePage + 1 - itemsPerPage} ${i18n.translate(
+			'to'
+		)} ${totalItems} ${i18n.translate('of')} ${totalItems} ${i18n.translate(
+			'entries'
+		)}.`}</p>
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/components/Table/Pagination/index.js:64]: Should be good to use the `i18n.sub`

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/components/Table/Pagination/index.js [hunk 2]
```diff
@@ -26,9 +27,9 @@ const TablePagination = ({
 }) => {
 	if (showDeltasDropDown || totalItems > itemsPerPage) {
 		const defaultLabels = {
-			paginationResults: 'Showing {0} to {1} of {2}',
-			perPageItems: '{0} items',
-			selectPerPageItems: '{0} items',
+			paginationResults: i18n.translate('showing-x-to-x-of-x'),
+			perPageItems: i18n.translate('x-items'),
+			selectPerPageItems: i18n.translate('x-items'),
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/components/Table/Pagination/index.js:32]: Hey @claricebrasil, just for clarify me, this is works integrate with `defaultLabels` for clay?

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

**Review by thaynam — APPROVED** [review]

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/components/Table/Pagination/index.js (hunk 1 above)
Comment (claricebrasil) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/components/Table/Pagination/index.js:64]: Nice tip, thank you!

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/components/Table/Pagination/index.js (hunk 2 above)
Comment (claricebrasil) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/components/Table/Pagination/index.js:32]: I didn't see any problems, but actually is not making any difference in the layout, so I leave the previous version without the translate.

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/CardSubscription/index.js [hunk 3]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @liferay/empty-line-between-elements */
```
[the remark this answers is not in the document]
Comment (claricebrasil) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/CardSubscription/index.js:1]: Sorry, I forgot!

**Review by jairmedeiros — APPROVED** [review]

**Comment by jairmedeiros** [comment]
ci:forward

**Pull request closed without merging**